src/components/guides/deploy-panel.tsx

```diff
@@ -0,0 +1,149 @@
+  return tour && <InformationButton onClick={tour.start} />;
+}
+
+export const DeployInformationPanel: FunctionComponent = () => {
+  const guideSteps: Array<ShepherdOptionsWithType> = [
+    introduction,
+    stepOne,
+    stepTwo,
+    stepThree,
+    stepFour,
+    stepComplete,
+  ];
+
+  const tourOptions: Tour.TourOptions = {
+    defaultStepOptions: {
+      cancelIcon: {
+        enabled: true,
+      },
+      scrollTo: {
+        behavior: "smooth",
+        block: "center",
+      },
+      popperOptions: {
+        modifiers: [{ name: "offset", options: { offset: [0, 12] } }],
+      },
+    },
+    useModalOverlay: true,
+  };
+  return (
+    <ShepherdTour steps={guideSteps} tourOptions={tourOptions}>
+      <Button />
+    </ShepherdTour>
+  );
+};
+
+const introduction = {
+  title: "Guide to Deploy Document Store",
+  text: `
+      <p>The document store is a smart contract on the Ethereum network that records the issuance and revocation status of OA documents.</p>
+      <p>In this guide, we will show you step by step instructions to deploy a new document store smart contract on the Ethereum.</p>
+  `,
+  buttons: [
+    {
+      classes: "w-full inline-flex justify-center text-sm font-medium bg-secondary-default hover:bg-secondary-hover ",
+      text: "Exit",
+      type: "cancel",
+    },
+    {
+      classes: "w-full inline-flex justify-center text-sm font-medium bg-primary-default hover:bg-primary-hover ",
+      text: "Start",
+      type: "next",
+    },
+  ],
+};
+
+const stepOne = {
+  title: "Guide to Deploy Document Store ( Step 1 )",
+  text: '<p>Click on the "Deploy New Document Store", This will open up dialog.</p>',
+  attachTo: { element: ".shepherd-deploy-modal-btn", on: "left" },
+  buttons: [
+    {
+      classes: "w-full inline-flex justify-center text-sm font-medium bg-secondary-default hover:bg-secondary-hover ",
+      text: "Back",
+      type: "back",
+    },
+  ],
+  advanceOn: { selector: ".shepherd-deploy-modal-btn", event: "click" },
+};
+
+const stepTwo = {
+  title: "Guide to Deploy Document Store ( Step 2 )",
+  text: `
+      <p>Enter your organisation name into the field.</p>
+      <p>Proceed to next step.</p>
+      `,
+  attachTo: { element: ".shepherd-organisation-txt", on: "right" },
+  buttons: [
+    {
+      classes: "w-full inline-flex justify-center text-sm font-medium bg-primary-default hover:bg-primary-hover ",
+      text: "Next",
+      type: "next",
+    },
+  ],
+};
+
+const stepThree = {
+  title: "Guide to Deploy Document Store ( Step 3 )",
+  text: '<p>Click on the "Deploy" button to start the process.</p>',
+  attachTo: { element: ".shepherd-deploy-btn", on: "left" },
+  buttons: [
+    {
+      classes: "w-full inline-flex justify-center text-sm font-medium bg-secondary-default hover:bg-secondary-hover ",
+      text: "Back",
+      type: "back",
+    },
+  ],
+  advanceOn: { selector: ".shepherd-deploy-btn", event: "click" },
+};
+
+const stepFour = {
```

Comment:
The deploy button is not disabled if there is no name. It means I can continue the guide but it won't successfully finished. Same for issuance and revoke (buttons are not disable if empty, we can just check for emptiness first)

Reply:
okay, added constraint that check if the field is empty before proceeding next step